### PR TITLE
fingertip mk1.1 added for testing

### PR DESCRIPTION
**Pull request by simeonedussoni**
Added:
- skingui conf for the mk1.1 fingertip
- device lines in yarpmanager
- conf files for yarprobotinterface for use w/ MC4+ board

all tested and working with the `Superbuild` and using the instructions here

### CODE REVIEW

**Review by pattacini — CHANGES_REQUESTED** [review]
I've one major request for change.
Then, I think @davidetome is a better fit as a reviewer.

@davidetome, let's remember to regenerate the docker image once merged.

File: test-skin-mc4plus/config-files/skin-fingertip.xml
Comment (simeonedussoni) [reply test-skin-mc4plus/config-files/skin-fingertip.xml:1]: actually the file skin-fingertip.xml is a brand new created for testing the mk1.1 fingertips.

Comment (pattacini) [reply test-skin-mc4plus/config-files/skin-fingertip.xml:1]: Ok! I thought the comment referred to the file [test-skin-mc4plus/config-files/applications/skin.xml](https://github.com/icub-tech-iit/test-skin-patches/pull/18/files/52464171d7559271bbd93e6bf74ab15fec4ea157#diff-6699960019c40cf4d4db37a11f7561d162fbc0e4a867972bb23cb2311ee21d96) for which I dropped my suggestion.

File: test-skin-mc4plus/config-files/applications/skin.xml
Comment (simeonedussoni) [reply test-skin-mc4plus/config-files/applications/skin.xml:1]: now there is a dedicated app for the fingertip testing, with the naming coherent to the configuration file i.e. with the `V2_2` suffix. 
In this way it will be straightforward to add another one for the 3DMID device.

Comment (pattacini) [reply test-skin-mc4plus/config-files/applications/skin.xml:1]: Ok, great 👍🏻 
But now we need to revert skin.xml to its previous state.

Comment (pattacini) [reply test-skin-mc4plus/config-files/applications/skin.xml:1]: Thanks!

**Review by pattacini — APPROVED** [review]
The PR is fine with me now 👍🏻 
Awaiting @davidetome's final approval before merging.

**Review by davidetome — APPROVED** [review]

**Pull request merged**
